### DISCUSSION TITLE
ESLint resolves all types as any when importing using Typescript's paths

### DISCUSSION THREAD

**Discussion by ValenCassa** [vercel/turborepo#6324] [Help]
Hey! I'm using the base template, only added `"baseUrl": "."` to the `tsconfig.json` file in the docs app. ESLint complains that everything that's being imported from the base path is `any` even tho TS picks up the correct types. 
![image](https://github.com/vercel/turbo/assets/37965406/1e7acdf1-dab8-4024-b78a-9402dce1ca54)

**Comment by anthonyshew** [comment]
It's likely that there's a misconfiguration in your typescript-eslint or ESLint setup.

Here's a pointer to the typescript-eslint documentation: https://typescript-eslint.io/linting/typed-linting/monorepos/
